src/sql/workbench/browser/modelComponents/declarativeTable.component.html

```diff
@@ -6,6 +6,7 @@
 				<th class="declarative-table-header" aria-sort="none" [style.width]="getColumnWidth(column)"
 					[ngStyle]="column.headerCssStyles" [attr.aria-label]="getHeaderAriaLabel(c)">
 					{{column.displayName}}
+					<span *ngIf="isRequired(c)" style="color:red;">*</span>
```

Comment:
As per adding the required indicator on the table header, I will be attending a11y office hours next Tuesday 7/20 for more insights on this issue. The migration extension might have more tables that could require the required indicator in the future.